src/test/java/seedu/address/testutil/TypicalMeetings.java

```diff
@@ -21,4 +21,13 @@
             .withDate(LocalDate.now())
             .withTime(LocalTime.now())
             .build();
+
+    public static Meeting[] getTypicalMeetings() {
```

Comment:
Can shift this to `PersonTest` because `getTypicalMeetings()` feels like I am retrieving all the meetings from `TypicalMeetings` but that is not the case here.

Reply:
I merely did this as `TypicalPersons` has a `getTypicalPersons` method. I suppose we could have a test equivalent of `SampleDataUtil` in `main`